go/pkg/cas/client.go

```diff
@@ -115,14 +120,22 @@ func DefaultClientConfig() ClientConfig {
 		// https://cloud.google.com/compute/docs/disks/optimizing-pd-performance#io-size
 		FileIOSize: 4 * 1024 * 1024, // 4MiB
 
-		FindMissingBlobsBatchSize: 1000,
-
-		BatchUpdateBlobsConcurrency: 256, // arbitrary
-
-		// This is a suggested approximate limit based on current RBE implementation for writes.
-		// Above that BatchUpdateBlobs calls start to exceed a typical minute timeout.
-		// This default might not be best for reads though.
-		MaxBatchTotalSizeBytes: 4000,
+		FindMissingBlobs: RPCConfig{
+			Concurrency: 64,
+			MaxItems:    1000,
+			Timeout:     time.Second,
```

Comment:
This may be too strict if we need to send RPC to opposite side of the earth?

Reply:
Changed to minute. The `client` package uses a minute, but for some reason I thought it is a second.